docs/.wordlist.txt

```diff
@@ -0,0 +1,76 @@
+Permalink
```

Comment:
is there a way to order this alphabetically? As it grows it gets really hard to find a specific word

Reply:
done